packages/grafana-data/src/types/datasource.ts

```diff
@@ -522,6 +522,10 @@ export interface DataQueryRequest<TQuery extends DataQuery = DataQuery> {
 
   // Make it possible to hide support queries from the inspector
   hideFromInspector?: boolean;
+
+  // For incremental queries that have modified the request, to let the backend know what the requested interval was
+  //@todo, store the delta? Or the stamp?
```

Comment:
I am storing the delta, logic being that this has value without knowing what the range is

Reply:
We could condense this value further by dividing the delta by the duration, giving us the expected number of samples "saved"